inspirehep/modules/multieditor/views.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2017 CERN.
+#
+# INSPIRE is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Module for backend of multi record editor used in http://inspirehep.net."""
+
+from __future__ import absolute_import, print_function, division
+
+from flask import Blueprint, request, jsonify, current_app, session
+
+from . import queries
+from . import actions
+
+
+blueprint = Blueprint(
+    'inspirehep_multieditor',
+    __name__,
+    url_prefix='/multieditor',
+)
+
+
+@blueprint.route("/update", methods=['POST'])
+def update():
+    """Basic view."""
+    user_actions = request.json['user_actions']
+    checked_ids = request.json['ids']
+    all_selected = request.json['allSelected']
+    searched_records = session.get('searched_records', [])
+    if searched_records:
+        ids = searched_records['ids']
+        index = searched_records['schema']
+    if all_selected:
+        ids = filter(lambda x: x not in checked_ids, ids)
```

Comment:
you have explained this to me a few times, but by reading the code is still not clear what `all_selected` means. I think you need a better naming.